### PR TITLE
Add Travis CI to run tests.

### PR DESCRIPTION
**Pull request by artem-zinnatullin**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+before_install:
+  - sudo apt-get update -qq
```
Comment (ming13) [inline .travis.yml]: `--quiet --quiet`?

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+  - sudo apt-get install -qq openssh-server
+  - mkdir -p ~/.ssh/
+  - chmod 700 ~/.ssh/
```
Comment (ming13) [inline .travis.yml]:
```
chmod u+rwx,go= ~/.ssh
```
Not sure if it is better, but for me personally it is easier to read — not everyone knows about their meaning.

File: .travis.yml (hunk 1 above)
Comment (artem-zinnatullin) [inline .travis.yml]: ok

File: .travis.yml (hunk 2 above)
Comment (artem-zinnatullin) [inline .travis.yml]: ok

File: .travis.yml [hunk 3]
```diff
@@ -0,0 +1,16 @@
+before_install:
+  - sudo apt-get update --quiet
+  - sudo apt-get install --quiet --yes openssh-server
+  - mkdir -p ~/.ssh/
+  - chmod u+rwx,go= ~/.ssh/
+  - ssh-keygen -b 2048 -t rsa -f ~/.ssh/id_rsa -q -N ""
+  - cp ~/.ssh/id_rsa.pub ~/.ssh/authorized_keys
+  - chmod u+rw,go= ~/.ssh/authorized_keys
+  - ssh-keyscan -t rsa localhost > ~/.ssh/known_hosts
+  - chmod 600 ~/.ssh/known_hosts
```
Comment (ming13) [inline .travis.yml]: Forgot this one.

Comment (artem-zinnatullin) [inline .travis.yml]: Foxed

**Pull request merged**